src/mono/wasm/debugger/BrowserDebugHost/Startup.cs

```diff
@@ -151,6 +151,12 @@ async Task ConnectProxy(HttpContext context)
                     }
 
                     var endpoint = new Uri($"ws://{devToolsHost.Authority}{context.Request.Path}");
+                    int runtimeId = 0;
+                    if (context.Request.Query.TryGetValue("RuntimeId", out var runtimeIdValue))
+                    {
+                        int.TryParse(runtimeIdValue.First(), out runtimeId);
```

Comment:
Check that `runtimeIdValue` is not empty, or use `FirstOrDefault()`.
Check the result of `TryParse`, unlikely though it might be.

Reply:
Maybe something like:

```csharp
                    int runtimeId = 0;
                    if (context.Request.Query.TryGetValue("RuntimeId", out StringValues runtimeIdValue) &&
                        int.TryParse(runtimeIdValue.FirstOrDefault(), out int parsedId);
                    {
					    runtimeId = parsedId;
                    }
```